src/admin/components/elements/ThumbnailCard/index.tsx

```diff
@@ -33,6 +33,7 @@ export const ThumbnailCard: React.FC<Props> = (props) => {
       className={classes}
       onClick={typeof onClick === 'function' ? onClick : undefined}
       onKeyDown={typeof onKeyDown === 'function' ? onKeyDown : undefined}
+      title={label ? label : typeof doc?.filename === 'string' ? doc?.filename : `[${t('untitled')}]`}
```

Comment:
We need to avoid using nested ternaries. Ultimately this should be an ESLint rule, but that's work for another day.